opm/autodiff/MultisegmentWell_impl.hpp

```diff
@@ -1897,7 +1897,7 @@ namespace Opm
     MultisegmentWell<TypeTag>::
     wellTestingPhysical(Simulator& simulator, const std::vector<double>& B_avg,
                         const double simulation_time, const int report_step, const bool terminal_output,
-                        WellState& well_state, WellTestState& welltest_state)
+                        WellState& well_state, WellTestState& welltest_state, wellhelpers::WellSwitchingLogger& logger)
```

Comment:
`logger` is unused, isn't it?

Reply:
True, none of the arguments are used here (they will be eventually in order to support WTEST for multisegment wells though). Wonder why I do not get warnings about the unused arguments?

Reply:
unused warnings do not always show up, kind of depending on the configurations.